config/testgrids/kubernetes/sig-cloud-provider/azure/config.yaml

```diff
@@ -3,7 +3,9 @@ dashboard_groups:
   dashboard_names:
     - provider-azure-master
     - provider-azure-on-call
+    - provider-azure-scalability
 
 dashboards:
 - name: provider-azure-master
 - name: provider-azure-on-call
+- name: provider-azure-scalability
```

Comment:
```suggestion
- name: provider-azure-1.16
```

Reply:
@feiskyer WDYT?

Reply:
Though kuberntes version is pinned to 1.16, we're only running the e2e tests on the latest version today. For this job, "scalability" should be included in the name. I'd prefer provider-azure-scalability

Reply:
@feiskyer -- this is actually the dashboard name, not the job name. What I'm suggesting is grouping the dashboards against the release version, instead of by the type of job. That way, you have a full picture of what's happening when you click on each dashboard.

Here's an example: https://testgrid.k8s.io/sig-release-1.17-blocking